onnxruntime/test/providers/tensorrt/tensorrt_basic_test.cc

```diff
@@ -595,65 +595,59 @@ TEST_P(TensorrtExecutionProviderCacheTest, Run) {
       }
     }
   } else if (cache_type.compare("timing") == 0) {
-    // add test code here
-
-    /* Following code block tests the functionality of engine and optimization profile of ORT TRT, including:
+    /* Following code block tests the functionality of timing cache, including:
```

Comment:
would it be better to just #ifdef out this entire block of code from line 597 to 653 rather than comment out individual lines?

Reply:
I think we still need to test whether the timing cache is existed when enabling timing cache.
ASSERT_TRUE(IsCacheExistedByType("./", ".timing"))